modules/thermo-cycler/production/firmware_uploader.py

```diff
@@ -19,8 +19,10 @@
 
 THIS_DIR = Path.cwd()
 DEFAULT_FW_FILE_PATH = THIS_DIR.joinpath('thermo-cycler-arduino.ino.bin')
-OPENTRONS_VID = 1240
-TC_BOOTLOADER_PID = 0xED12
+OPENTRONS_VID       = 1240  # 0x04D8
```

Comment:
You can write numbers in hex in python, and I feel like you probably should

Reply:
I switched to using decimal because the comport function that lists the available ports and their pid/vids returns those numbers in decimal; it's just easier to compare decimal to decimal. Also I was getting an error when trying to convert the values returned by `.pid`, `.vid` to hex. It was quite some time ago so I don't remember the error. I'll give it another try